features/restaurant_grouping.feature

```diff
@@ -0,0 +1,21 @@
+Feature: User can select restaurant by category
+  As a user
+  In order to get choose what to eat
+  I would like to see a restaurant by category
+
+  Background:
+    Given the following restaurants exist
+      | name       | address       |
+      | My Thai    | Stockholm     |
+      | My Mexican | Mexico city   |
```

Comment:
If you reverse the order of these steps you should be able to assign a category to the restaurants you create